core/src/main/java/org/mapstruct/InheritInverseConfiguration.java

```diff
@@ -22,6 +22,54 @@
  * If more than one matching inverse method exists, the name of the method to inherit the configuration from must be
  * specified via {@link #name()}
  *
+ * <p>
+ * <strong>Example</strong>
+ * </p>
+ *
+ * <pre>
+ * <code>
+ * &#64;Mapper
+ * public interface HumanMapper {
+ *
+ *     Human toHuman(HumanDto humanDto);
+ *
+ *     &#64;InheritInverseConfiguration
+ *     HumanDto toHumanDto(Human human);
+ * }
+ *
+ * // generates
+ * public class HumanMapperImpl implements HumanMapper {
+ *
+ *     &#64;Override
+ *     public Human toHuman(HumanDto humanDto) {
+ *         if ( humanDto == null ) {
+ *             return null;
+ *         }
+ *
+ *         Human human = new Human();
+ *
```

Comment:
perhaps better to condense the code a bit more (I know there are empty lines in the generated code for readability purposes)